meta-multimedia/recipes-multimedia/libcamera/libcamera.bb

```diff
@@ -0,0 +1,20 @@
+SUMMARY = "Linux libcamera framework"
+SECTION = "libs"
+
+LICENSE = "GPL-2.0 & LGPL-2.1"
+LIC_FILES_CHKSUM = "\
+    file://licenses/gnu-gpl-2.0.txt;md5=b234ee4d69f5fce4486a80fdaf4a4263 \
+    file://licenses/gnu-lgpl-2.1.txt;md5=4b54a1fd55a448865a0b32d41598759d \
+"
+
+SRC_URI = "git://linuxtv.org/libcamera.git;protocol=git"
+
+S = "${WORKDIR}/git"
+SRCREV = "${AUTOREV}"
+
+DEPENDS = "python udev qtbase qtbase-native"
```

Comment:
turn qt dependencies into PACKAGECONFIG and keep it off by default

Reply:
qt dependency is for run time and not during the compiling. So we have removed it, as it is needed only for RaspberryPi config.

Reply:
> turn qt dependencies into PACKAGECONFIG and keep it off by default

By default, we had made the qt dependency off, and enabled once the DISTRO_FEATURES get appended with the "qt".